### ISSUE TITLE
IMPORTANT! Accessibility WCAG 2.4.7 Focus in elements is not visible 

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#145]
Elements in Header, Navigation, Tabs, some Form-Elements etc.
In fact it’s a easy to correct:
The pseudo classes :focus should have the same attribute like :hover  
(Already well done with text-links)
